### DISCUSSION TITLE
Idea: Selective SSR

### DISCUSSION THREAD

**Discussion by kasvith** [withastro/roadmap#366] [Proposal]
Currently SSR just bundling Astro components to be served over some adapter. 
It doesn't support prerendering in this mode, it's kinda bad in my opinion.

There can be certain routes which can be pre-rendered on build time,
There can be some routes which need dynamic data with SSR every time

Astro needs a way to let users determine which routes can be pre-rendered even with SSR

**Comment by delucis** [comment]
We have a draft RFC up for this feature at the moment! https://github.com/withastro/rfcs/pull/362

**Comment by florian-lefebvre** [comment]
Closing because this is outdated
